pcbdraw/pcbdraw.py

```diff
@@ -867,6 +949,24 @@ def main():
         print(e)
         sys.exit(1)
 
+    # If some keys aren't in the loaded style compared to the default style, copy it from the default style
+    for s in default_style:
+        if s not in style:
+            style[s] = default_style[s]
```

Comment:
If we inherit properties from default styles, we should also check that there are no extra keys in the user-supplied style. When we don't check for them it is really hard to spot typos in key names (since it silently fails)